cogs/automation/intervally/daily_auto.py

```diff
@@ -18,9 +18,115 @@
     datetime.time(hour=0, minute=5, tzinfo=datetime.timezone.utc)
 ]
 
+REMINDER_MESSAGES = {
+    datetime.time(hour=8, minute=30, tzinfo=datetime.timezone.utc): f"Hi {{mention}}, it's your turn to write the editorial today! Please complete it by 6pm to help other community members in need! {Assets.blob_victory}",
+    datetime.time(hour=17, minute=43, tzinfo=datetime.timezone.utc): f"Hi {{mention}}, remember to turn in your editorial by 6PM today! {Assets.blob_maman}",
+    datetime.time(hour=18, minute=30, tzinfo=datetime.timezone.utc): f"Hi {{mention}}, please turn in your editorial immediately! If you need help, please contact Community members for help! {Assets.blob_taco}"
+}
+
+
+class DailyReminder(commands.Cog):
+    def __init__(self, client):
+        self.client = client
+        self.task_completed = False
+        self.reminder_task.start()
+        self.daily_thread = None
+        self.problem = None
+        self.assignee_id = None
+
+    @tasks.loop(minutes=1)
+    async def reminder_task(self):
+        now = datetime.datetime.now()
+        for reminder_time in REMINDER_MESSAGES.keys():
```

Comment:
Đoạn code này chưa sạch và tối ưu đâu nhé. Vì tần suất lặp lại của task này rất lớn (1p/1 lần), nên ta cần có logic để return luôn nếu reminder time không nằm trong map này.

Typically mình sẽ tạo một object datetime chỉ sử dụng `hour` và `minute` của now và sử dụng `not in` để `return` khỏi hàm, trc khi thực hiện các tác vụ tiếp theo.